Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinateConverter.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using UnityEngine;
+
+namespace AWSIM.Geographic
+{
+public static class GeoCoordinateConverter
```

Comment:
Please add a brief comment on the calculation method used in this class. For example, post the URL of the site you referred to.